WordPress/Classes/ViewRelated/Reader/ReaderStreamViewController.swift

```diff
@@ -210,6 +206,9 @@ import WordPressComAnalytics
         // pull to refresh animation.
         view.userInteractionEnabled = readerTopic != nil
 
+        NSNotificationCenter.defaultCenter().addObserver(self, selector: #selector(defaultAccountDidChange(_:)), name: WPAccountDefaultWordPressComAccountChangedNotification, object: nil)
```

Comment:
Hmm, maybe it would be best to observe it anyway? Maybe not make an assumption that it will never change while the VC is allocated.